causalpy/pymc_models.py

```diff
@@ -85,10 +204,52 @@ def score(self, X, y) -> pd.Series:
 
 
 class WeightedSumFitter(ModelBuilder):
-    """Used for synthetic control experiments"""
+    """
+    Used for synthetic control experiments
+
+    .. note: Generally, the `.fit()` method should be rather than calling
+    `.build_model()` directly.
+
+    Defines the PyMC model:
+
+    .. math::
+
+        sigma \sim HalfNormal(1)
+
+        beta \sim Dirichlet(1,...,1)
+
+        mu = X * beta
+
+        y \sim Normal(mu, sigma)
+
+    Example
+    --------
+    >>> import causalpy as cp
+    >>> import numpy as np
+    >>> from causalpy.pymc_models import WeightedSumFitter
+    >>> sc = cp.load_data("sc")
+    >>> X = sc[['a', 'b', 'c', 'd', 'e', 'f', 'g']]
+    >>> y = np.asarray(sc['actual']).reshape((sc.shape[0], 1))
+    >>> wsf = WeightedSumFitter(sample_kwargs={"progressbar": False})
+    >>> wsf.fit(X,y)
+    Inference ...
+    """  # noqa: W605
 
     def build_model(self, X, y, coords):
-        """Defines the PyMC model"""
+        """
+        Defines the PyMC model:
+
+        .. math::
+
+            sigma \sim HalfNormal(1)
+
+            beta \sim Dirichlet(1,...,1)
+
+            mu = X * beta
+
+            y \sim Normal(mu, sigma)
```

Comment:
Delete. I think we only need the model maths once, and the best place for that is in the main class docstring, we don't need to repeat it for the `build_model` method. Same comment goes for the other classes